pkgs/top-level/all-packages.nix

```
@@ -13517,6 +13517,9 @@ with pkgs;
 
   remarkable2-toolchain = callPackage ../development/tools/misc/remarkable/remarkable2-toolchain { };
 
+  spicedb     = callPackage ../servers/spicedb { };
```

```suggestion
  spicedb = callPackage ../servers/spicedb { };
```
Maybe it is not necessary to align the equal sign, just a suggestion.